models/prices/arbitrum/prices_arbitrum_tokens.sql

```diff
@@ -197,6 +197,6 @@ FROM
     ('gyd-gyro-dollar','arbitrum','GYD',0xca5d8f8a8d49439357d3cf46ca2e720702f132b8, 18),
     ('a51-a51-finance','arbitrum','A51',0xb3f13b0c61d65d67d7d6215d70c89533ee567a91, 18),
     ('oseth-stakewise-staked-eth','arbitrum','osETH',0xf7d4e7273e5015c96728a6b02f31c505ee184603, 18),
-    -- ('sliz-solidlizard-dex-token','arbitrum','SLIZ',0x463913D3a3D3D291667D53B8325c598Eb88D3B0e, 18), -- Inactive
+    ('sliz-solidlizard-dex-token','arbitrum','SLIZ',0x463913D3a3D3D291667D53B8325c598Eb88D3B0e, 18),
```

Comment:
I am not sure but it shows ` "is_active": false,` in my view:
https://api.coinpaprika.com/v1/coins/sliz-solidlizard-dex-token

```
{"id":"sliz-solidlizard-dex-token","name":"SolidLizard Dex Token","symbol":"SLIZ","rank":0,"is_new":false,"is_active":false,"type":"token","contract":"0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","platform":"arb-arbitrum","contracts":[{"contract":"0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","platform":"arb-arbitrum","type":"Other"}],"logo":"https://static.coinpaprika.com/coin/sliz-solidlizard-dex-token/logo.png","parent":{"id":"arb-arbitrum","name":"Arbitrum","symbol":"ARB"},"tags":[{"id":"arbitrum-ecosystem","name":"Arbitrum Ecosystem","coin_counter":624,"ico_counter":1},{"id":"decentralized-exchange-dex","name":"Decentralized Exchange (DEX)","coin_counter":148,"ico_counter":0}],"team":[],"description":"","message":null,"open_source":false,"started_at":null,"development_status":null,"hardware_wallet":false,"proof_type":null,"org_structure":null,"hash_algorithm":null,"links":{"explorer":["https://arbiscan.io/token/0x463913D3a3D3D291667D53B8325c598Eb88D3B0e"],"medium":["https://medium.com/@solidlizardfi"],"website":["https://solidlizard.finance"]},"links_extended":[{"url":"https://discord.com/invite/fSdcAQcGxw","type":"discord"},{"url":"https://arbiscan.io/token/0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","type":"explorer"},{"url":"https://medium.com/@solidlizardfi","type":"medium"},{"url":"https://t.me/solidlizardfi","type":"telegram","stats":{"members":500}},{"url":"https://twitter.com/solidlizardfi","type":"twitter","stats":{"followers":7517}},{"url":"https://solidlizard.finance","type":"website"}],"whitepaper":{"link":"https://solidlizard.gitbook.io/solidlizard/solidlizard-protocol/welcome","thumbnail":"https://static.coinpaprika.com/storage/cdn/whitepapers/11037299.jpg"},"first_data_at":"2023-03-19T00:00:00Z","last_data_at":"2024-06-24T04:23:00Z"}
```

Maybe it is not very stable.

Reply:
Ohhh okay! What can be done in such cases?
Will it create any issue in future?
@Hosuke

Reply:
I think we can add this token, should be fine.

Reply:
Thanks a lot!!